### PR TITLE
UI

### PR DESCRIPTION
**Pull request by hakichris**
In this branch, we created a user interface for the school library app

- [x] no linters error 
- [x] UI added 
- [x]  app and main class added

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    @rental << Rental.new(date, @person[person_id], @book[book_id])
+    puts 'Rental created successfully'
+  end
+
+  def list_all_rentals
+    print 'ID of person: '
+    id = gets.chomp.to_i
+
+    rentals = @rental.filter { |rental| rental.person.id == id }
```
Comment (codecaiine) [inline app.rb:110]: - [ ] Good job so far 👏  But I have an issue when I am trying to list a rental and select a person id as you can see on the screenshot below ⬇️  Please fix it in order to make your app works as expected 👍 

RUBYYY

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @hakichris , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good UI setup setup ✔️
- No rubocop errors ✔️
-  Good use of gitflow ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (hakichris) [reply app.rb:110]: @codecaiine problem fixed, thank you

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    @person.each_with_index do |person, index|
+      puts "#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+
+    person_id = gets.chomp.to_i
+
+    print 'Date: mm/dd/yy '
+    date = gets.chomp
+
+    @rental << Rental.new(date, @person[person_id], @book[book_id])
```
Comment (codecaiine) [inline app.rb:102]: - [ ] Good job so far 👏  But while you were fixing List a rental by selecting a person id but creating Rental which was working, DOES NOT work anymore makes the rentals empty when you are selecting student ID . Please make sure to fix all the errors and pay attention when you are fixing errors to not affect other functions and make your app work as expected  😄 

rental

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @hakichris , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good UI setup setup ✔️
- No rubocop errors ✔️
- Good use of gitflow ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 2 above)
Comment (hakichris) [reply app.rb:102]: Hello @codecaiine, I don't see this error on my end. Would you like to meet and check the problem together? I am texting you on Slack

**Review by codecaiine — APPROVED** [review]
Hi @hakichris , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good UI setup ✔️ 
- App works as expected  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**